common/djangoapps/third_party_auth/tests/test_pipeline_integration.py

```diff
@@ -304,9 +305,6 @@ def test_quarantine(self):
 class EnsureUserInformationTestCase(testutil.TestCase, test.TestCase):
     """Tests ensuring that we have the necessary user information to proceed with the pipeline."""
 
-    def setUp(self):
-        super(EnsureUserInformationTestCase, self).setUp()
-
```

Comment:
@UmanShahzad Why this change? (genuine question)

Reply:
Drive-by change when I was scrolling down to write my own test case. This setup does absolutely nothing :p